solutions/community-deploy/cloud/AWS/README.md

```diff
@@ -1,37 +1,30 @@
 # Build & Deploy CORTX Stack on Amazon Web Services 
 
-This document discusses the procedure to compile the CORTX Stack and deploy it on AWS instance.
+This document discusses the procedure to compile the CORTX Stack and deploy on AWS environment (One/Multi-node)
 
+**Prerequisites:**
 
-## Prerequisite 
-
-Ensure that you have an AWS account with Secret Key and Access Key.
-
-## Install and setup Terraform and AWS CLI
-
-- Clone the `cortx-re` repository and change the directory to `cortx-re/solutions/community-deploy/cloud/AWS`.
-```
-git clone https://github.com/Seagate/cortx-re && cd $PWD/cortx-re/solutions/community-deploy/cloud/AWS
-```
-
-- Install the required tools.
+- Ensure that you have an AWS account with Secret Key and Access Key.
+- Install the required tools on the local host
 ```
 ./tool_setup.sh
```

Comment:
We need to clone cortx-re repo before executing `./tool_setup.sh` . https://github.com/Seagate/cortx-re/blob/mukul-multinode-2/solutions/community-deploy/cloud/AWS/tool_setup.sh 

https://github.com/Seagate/cortx-re/tree/mukul-multinode-2/solutions/community-deploy/cloud/AWS

![image](https://user-images.githubusercontent.com/63289460/183651449-68063a95-b1b5-4224-b990-9bbb82f4b573.png)

Reply:
fixed